source/blood/src/gamemenu.cpp

```diff
@@ -182,25 +194,64 @@ void CGameMenuMgr::Draw(void)
 
         m_mousewake_watchpoint = 0;
     }
+
+    vec2_t cursorpos = { (m_mousepos.x >> 16) + 9, (m_mousepos.y >> 16) + 18};
+
     // Display the mouse cursor, except on touch devices.
-    if (MOUSEACTIVECONDITION)
+    if (MOUSEACTIVECONDITION && !m_bFirstPush)
     {
-        int32_t a = kCrosshairTile;
+        int16_t mousetile = 3522; // disembodied arm
```

Comment:
Leave crosshair variant for now, i'm going to add more cursor variants later

Reply:
What about tile 1043 (red arrow pointing up)? And rotate a little so it would look similar as in BloodGDX, just maybe not that big and different color. Just an idea.

Reply:
I like this variant

Reply:
By "this" you mean the red arrow or the aim reticle?

Reply:
red arrow